code/modules/mob/living/silicon/robot/life.dm

```diff
@@ -194,7 +113,7 @@
 		IgniteMob()
 
 /mob/living/silicon/robot/update_canmove()
-	if(paralysis || stunned || weakened || buckled || lockcharge)
+	if(stat || buckled || lockcharge)

```

Comment:
stunned,weakened and paralysis always mean unconsciousness for borgs, and since unpowered no longer use unconsciousness we can replace all three by stat.